api/tests/src/api/users/test_user_route_token.py

```diff
@@ -25,3 +28,30 @@ def test_post_user_route_token_400(client, api_auth_token):
     resp = client.post("v1/users/token", headers={"X-Auth": api_auth_token})
     assert resp.status_code == 400
     assert resp.get_json()["message"] == "Missing X-OAuth-login-gov header"
+
+
+def test_post_user_route_token_logout_200(
+    enable_factory_create, client, db_session, api_auth_token
+):
+    user = UserFactory.create()
+    token, _ = create_jwt_for_user(user, db_session)
+    db_session.commit()
+
+    resp = client.post("v1/users/token/logout", headers={"X-SGG-Token": token})
+
+    assert resp.status_code == 200
+
```

Comment:
I'd check that the record in the DB is updated afterwards.

 Something like:
```
token, user_token_session = create_jwt_for_user(user, db_session)
...
db_session.refresh(user_token_session)
assert user_token_session.is_valid is False
```

Reply:
Added